### DISCUSSION TITLE
[New Component] [contextmenu] 希望支持contextmenu,也就是右键菜单

### DISCUSSION THREAD

**Discussion by Cashew-wood** [element-plus/element-plus#9225] [Feature Request]
Feature type: **`New Component`**

## Details

- New component name: `el-contextmenu`

## Description

右键菜单以及子菜单，同时建议增加dropdown的子菜单支持

### Additional comments

_(empty)_

**Comment by Drinjs** [comment]
自从我开始使用element-ui以来，始终觉得右键菜单对于element-xx不可或缺，不知道大佬们怎么认为
